cargo-quickinstall/src/lib.rs

```diff
@@ -227,16 +306,18 @@ pub fn curl_json(url: &str) -> Result<JsonValue, InstallError> {
         })
 }
 
+fn get_quickinstall_download_url(crate_name: &str, version: &str, target: &str) -> String {
```

Comment:
let's try to avoid functions that take a bunch of stringly typed arguments: it's really easy to get the order mixed up. Just pass in `crate_details: &CrateDetails` or something.

Reply:
Done!